packer/build_platinum_ami_amzn2.json

```diff
@@ -0,0 +1,39 @@
+{
+  "variables": {
+    "source_ami": "",
+    "vault_password_file": "",
+    "subnet_id": "subnet-81ecfbab"
+  },
+  "builders": [{
+    "type": "amazon-ebs",
+    "ssh_username": "ec2-user",
+    "instance_type": "m4.large",
+    "region": "us-east-1",
+    "subnet_id": "{{user `subnet_id`}}",
+    "associate_public_ip_address": "false",
+    "source_ami": "{{user `source_ami`}}",
+    "ami_name": "bb2-amzn2-platinum-{{isotime \"20060102030405\"}}",
+    "ssh_pty": true,
+    "launch_block_device_mappings": [
+      {
+        "device_name": "/dev/xvda",
+        "volume_type": "gp2",
+        "volume_size": 30,
+        "delete_on_termination": true
+      }
```

Comment:
Amazon Linux 2 gold image has a 8GB root volume by default, we need to expand that for log space.